scripts/plugins.sh

```diff
@@ -3,6 +3,16 @@
 # This script builds the required plugins.
 set -e
 
+if [[ -z "$GOOS" ]]; then
```

Comment:
I need the `GOOS` and `GOARCH` to construct the file name correct for embedding. 
This means as the current script is run those need to be explicitly set when running, this means when running `make dev-with-plugins` `GOOS` and `GOARCH` need to be redundantly set i.e. `GOOS=darwin GOARCH=amd64 make dev-with-plugins`
If there is another way to get OS and ARCH let me know!